ports/nodesoup/portfile.cmake

```
@@ -0,0 +1,23 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO olvb/nodesoup
+    REF 3158ad0
+    SHA512 be98cd5a1106fb1b6e6cb6b880229f590c2d4c4cc176dcceb2e2226ff3f2344ccb4510fb3a0911e9329701af50f076ee2efb9a3afc9e985b4d9c3fb92c12102d
+    HEAD_REF main
```

```suggestion
    HEAD_REF master
```
Be consistent with upstream.